### PR TITLE
Input test and exceptions

### PR DESCRIPTION
**Pull request by svj0hn**
Improves on Issue #21 - please review.

### CODE REVIEW

File: src/test/java/tests/PigLatinConverterTest.java [hunk 1]
```diff
@@ -51,4 +54,17 @@ public void testConvertStringToPigLatinMultipleWords() {
...
+	public void testGetInputFromCommand() {
+		String input = "some#anNoYING%%str!ng";
+		InputStream stdin = System.in;
+		
+		InputStream in = new ByteArrayInputStream(input.getBytes());
+		System.setIn(in); // Spoof user input
+		String output = PigLatinConverter.getInputFromCommand();
+		System.setIn(stdin);
+		
+		assertEquals(output, "some#anNoYING%%str!ng");
```
Comment (antekillen) [inline src/test/java/tests/PigLatinConverterTest.java]: Here you should assertEquals(output, input)

File: src/main/java/piglatin/PigLatinConverter.java [hunk 2]
```diff
@@ -13,7 +13,17 @@ public static boolean isVowel(char letter) {
...
+		
+		try {
+			input = scan.nextLine();
+		}
+		catch (java.util.NoSuchElementException e) {
+			System.out.println("NoSuchElementException occured while scanning input!");
+		}
+		catch (java.lang.IllegalStateException e) {
+			System.out.println("IllegalStateException occured while scanning input!");
+		}
```
Comment (antekillen) [inline src/main/java/piglatin/PigLatinConverter.java]: Please format the above as

try {

} catch (ExceptionType name) {

} catch (ExceptionType name) {

}

**Review by antekillen — COMMENTED** [review]
Looks fine. Only formatting comments.

**Comment by svj0hn** [comment]
Updated according to @antekillen 's requests.

@martinlindf , feel free to merge if you think it looks ok.

**Comment by martinlindf** [comment]
Seems good. Merged.

**Pull request merged**